src/Mjolnir.ts

```diff
@@ -658,4 +666,12 @@ export class Mjolnir {
             message: message /* If `undefined`, we'll use Synapse's default message. */
         });
     }
+
+    public queueRedactUserMessagesIn(userId: string, roomId: string) {
+        this.eventRedactionQueue.add(new RedactUserInRoom(userId, roomId));
+    }
+
+    public async processRedactionQueue(roomId?: string) {
```

Comment:
As usual, please document :)

In particular, you should clarify that if `roomId` is not specified, it processes redaction queues in all rooms.